src/nix/doctor.cc

```diff
@@ -130,6 +135,15 @@ struct CmdDoctor : StoreCommand
 
         return checkPass("Client protocol matches store protocol.");
     }
+
+    void checkTrustedUser(ref<Store> store)
+    {
+        bool trusted = store->isTrustedUser();
+        if (trusted)
+            checkInfo("You are a trusted-user on store uri: " + store->getUri());
```

Comment:
Do we need to print the store uri here? It doesn't seem very useful (or if we want to print it, we could have it as its own line somewhere else)

Reply:
I think it's useful for debug, especially in doctor. If you do `nix doctor` and my store is a remote store, I want to know. It helps me debug my configuration. Granted this could also be done by increasing Nix verbosity with  `-v`. My preference is to keep printing it.